services/spar/test-integration/Test/Spar/Scim/UserSpec.hs

```diff
@@ -2112,6 +2113,27 @@ specSCIMManaged = do
           updateProfileBrig brig uid uupd !!! do
             (fmap Wai.label . responseJsonEither @Wai.Error) === const (Right "managed-by-scim")
             statusCode === const 403
+    it "created_on should be filled in CSV export" $ do
```

Comment:
There is a test `testListTeamMembersCsv` in galley that (I think) could be easily duplicated into a variant that generates the members via scim, not team invitations.  That test should fail before the changes in this PR, and succeed after.

If you don't want to move this test there and avoid code redundancy, please cross-reference the two tests, that's just as good IMO.